ui/job-view/headerbars/SecondaryNavBar.jsx

```diff
@@ -283,11 +284,8 @@ class SecondaryNavBar extends React.PureComponent {
               } ${!duplicateJobsVisible ? 'strikethrough' : ''}`}
               tabIndex="0"
               role="button"
-              title={
-                duplicateJobsVisible
-                  ? 'Hide duplicate jobs'
-                  : 'Show duplicate jobs'
-              }
+              title={toggleDuplicateJobsTitle}
+              label={toggleDuplicateJobsTitle}
```

Comment:
This might also be best changed to the react-strap Button component if it doesn't change the appearance drastically.

Reply:
Perhaps you could take a screenshot of how it looks with a button and @camd can give it an OK or not.

Reply:
Fixed, [here](https://srv2.imgonline.com.ua/result_img/imgonline-com-ua-2to1-eAB6aDx81slZL4Op.png) is screenshot with before/after.

Reply:
The screenshot link is showing a 404. Next time you can drag a screenshot into a comment and it will embed it here - no need to link to a third party site :)